### PR TITLE
Join table for visits

### PR DESCRIPTION
**Pull request by Jawad-Nazari**
### In this PR, I did the following:

-  Created a table named vets.
- Added many-to-many relationship between the species and vets tables .
- Added many-to-many relationship between the animals and vets tables .
- Inserted data to visits, vets, and specializations as per project requirements.

Who was the last animal seen by William Tatcher?
q1

How many different animals did Stephanie Mendez see?
q2

List all vets and their specialties, including vets with no specialties.
q3

List all animals that visited Stephanie Mendez between April 1st and August 30th, 2020.
q4

What animal has the most visits to vets?
q5

Who was Maisy Smith's first visit?
q6

Details for most recent visit: animal information, vet information, and date of visit.
q7

How many visits were with a vet that did not specialize in that animal's species?
q8

What specialty should Maisy Smith consider getting? Look for the species she gets the most.
q9

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -129,3 +129,73 @@ WHERE owners.full_name = 'Dean Winchester' AND animals.escape_attempts = 0;
...
+ORDER BY visits.visit_date DESC LIMIT 1;
+
+
+SELECT COUNT(*) AS num_of_visits FROM visits
+JOIN animals ON visits.animal_id = animals.id
+JOIN vets ON visits.vet_id = vets.id
+JOIN specializations ON vets.id = specializations.vet_id
+JOIN species ON animals.species_id = species.id
+WHERE species.id != specializations.species_id;
+
```
Comment (laiifuu) [inline queries.sql]: The answer for this query should be 12 visits, which include all visits by Maisy Smith, 2 visits by William Tatcher and 1 visit by Jack Harkness. And your screenshot shows 7.

File: queries.sql [hunk 2]
```diff
@@ -129,3 +129,73 @@ WHERE owners.full_name = 'Dean Winchester' AND animals.escape_attempts = 0;
...
+
+SELECT species.name AS Specialty from visits
+JOIN animals ON visits.animal_id = animals.id
+JOIN vets ON visits.vet_id = vets.id
+JOIN specializations ON vets.id = specializations.vet_id
+JOIN species ON animals.species_id = species.id
+WHERE vets.name = 'Maisy Smith'
+GROUP BY species.name ORDER BY COUNT(*) DESC LIMIT 1;
```
Comment (laiifuu) [inline queries.sql]: The response for this query should be `Digimon` as she has 6 visits by pets of this type. But your screenshot shows `0 rows`.

**Review by laiifuu — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED :recycle: 

## Hi @Jawad-Nazari :wave: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

so-close


## Highlights
- [x] Organized queries :heavy_check_mark: 
- [x] Great take on data insertion :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review :point_down:_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (Jawad-Nazari) [reply queries.sql]: Hi @laiifuu ,
Thanks for reviewing my code. I have updated my query to get the correct result. 🙏

File: queries.sql (hunk 2 above)
Comment (Jawad-Nazari) [reply queries.sql]: Hi @laiifuu ,

Thanks for reviewing my code. I have updated my query to reflect the correct answer. 🙏

**Review by Deepakdanger — APPROVED** [review]
Hi @Jawad-Nazari , 

### Status: Approved ✔️

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚢 
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**